mmdet/apis/export.py

```diff
@@ -199,7 +200,25 @@ def export_to_openvino(cfg, onnx_model_path, output_dir_path, input_shape=None,
         run(command_line, check=True)
 
 
-def optimize_onnx_graph(onnx_model_path):
+def optimize_onnx_graph(onnx_model_path, precision='FP32'):
+    def convert_np_to_float16(np_array, min_positive_val=1e-7, max_finite_val=1e4):
+        def between(a, b, c):
+            return np.logical_and(a < b, b < c)
+        np_array = np.where(between(0, np_array, min_positive_val), min_positive_val, np_array)
+        np_array = np.where(between(-min_positive_val, np_array, 0), -min_positive_val, np_array)
+        np_array = np.where(between(max_finite_val, np_array, float('inf')), max_finite_val, np_array)
+        np_array = np.where(between(float('-inf'), np_array, -max_finite_val), -max_finite_val, np_array)
```

Comment:
Do I understand that correct that this is similar to:
```suggestion
        np_array = np.sign(np_array) * np.clip(np.abs(np_array), min_positive_val, max_finite_val))
```